README.md

```diff
@@ -46,6 +46,29 @@ a system CA bundle is unavailable.
 [openssl-probe]: https://github.com/alexcrichton/openssl-probe
 [webpki-roots]: https://github.com/rustls/webpki-roots
 
+## Deployment Considerations
+
+When choosing to use `rustls-platform-verifier` or another trust store option, there are important differences to consider. These
+are primarily about root certificate availability:
+
+| Backend                          | Updates                         | Roots used                                                                                            |
+|----------------------------------|---------------------------------|-------------------------------------------------------------------------------------------------------|
+| OS/platform (non-Linux/BSD)      | Live pushes                     | System store, with correct (dis)trust decisions from every source available.                          |
+| `rustls-native-certs` + `webpki` | Live pushes                     | System store, with no (dis)trust decisions. All roots are treated equally regardless of their status. |
+| `webpki-roots` + `webpki`        | Static, manual updates required | Hardcoded Mozilla CA roots, and all roots are treated equally.                                        |
+
+**In general**: It is the opinion of the `rustls` team and platform verifier maintainers that this is the best default available for client-side libraries and applications
+making connections to TLS servers when running on common operating systems. This is because it gets both live trust information (new roots, explicit markers, and auto-managed CRLs)
+and better matches the common expectation of apps running on that platform (to use proxies, for example). Otherwise, it becomes your maintenance burden to
+ship updates right away in order to handle increasing numbers of positive and negative trust events in the WebPKI/certificate ecosystem, or risk availability and security concerns.
+
+Even though platform verifiers are sometimes implemented in memory-unsafe languages, it is very unlikely that Rust apps using this library will become a point of weakness.
+This is due to either using a smaller set of servers or just being less exposed then other critical functions of the operating system, default web browser, etc.
+But if you are, for example, fuzzing or scanning all certificates on the open internet, using a 100% Rust option like `webpki` is a better and more secure option.
```

Comment:
I'm not sure I follow the fuzzing angle. I would expect anyone fuzzing is doing so with the intent to find bugs, and so switching to a backend that's less likely to have bugs seems counter-productive. Maybe strike that part of the wording and emphasize a use-case where you're dealing with a large number of untrusted inputs like the "scan the internet" usecase mentioned?

Reply:
I think I might have used the word "fuzzing" too ambiguously here. I wasn't trying to refer to the verifier backend but an activity where, like you guessed, you are going to be processing a lot of untrusted TLS data and/or certificates for an experiment. For example:
- Seeing how many TLS servers `rustls` with a specific configuration can connect to
- Harvesting data from various untrusted TLS endpoints exposed on the internet
- Extracting info from a known-evil endpoint (C2 server, etc)

I can try and emphasize a use case but I did want it to be a generic "dont go poking known-evil things with this."

Reply:
> I wasn't trying to refer to the verifier backend but an activity where, like you guessed, you are going to be processing a lot of untrusted TLS data and/or certificates for an experiment. For example:

That makes sense, but the examples you list here all seem close to the category of "scanning all certificates on the open internet". Does having the "fuzzing" part of the text add much with respect to this thought or is it too ambiguous to be clear?

I still lean towards thinking it would be clearer without but I don't feel very strongly.

Reply:
> Does having the "fuzzing" part of the text add much with respect to this thought or is it too ambiguous to be clear?

With a pair of fresh eyes I think so, yes. After reading the existing text and comparing it to how I wrote it in this thread the problem became more apparent. I think its best to remove it and just imply "fuzz-like" behavior from the listed examples instead that readers can pick up.

Please have a look at the section again, if you don't mind. I rewrote it to try and be more clear about what constitutes an untrusted use case, borrowed some of your wording, and formatted it bullets to be easier to read/scan.

Reply:
The revisions look great, thank you!